Решение заданий урока 9

SlavaGrishanin opened the pull request.

SlavaGrishanin committed:
Добавление NoteContentDisplayActivity

SlavaGrishanin committed:
Исправление макетов: у элементов ConstraintLayout занчение wrap_content атрибута layout_width замененр на 0dp; добавление отсутствующих constraint'ов

primus-anonymous commented on app/src/main/java/GeekBrians/Slava_5655380/Note/NotesDAO.java:
```diff
@@ -17,7 +17,7 @@ public NotesDAO() {
                                     "Описание первой заметки"
                             ),
-                            "Содержимое первой заметки"));
+                            "Сооооооооооооооооооооооооооооооооооооооооооооооооооооооооооооооооооооооооооооооооооооооооооооооооооооооооооооооооооооооооооооооооооооооооооооооооооооооооооооооооооооооооооооооооооооооооооооооооооооооооооооооооооооооооооооооооооооооооооооооооооооооооооооооооооооооооооооооооооооооооооооооооооооооооооооооооооооооооооооооооооооооооооооооооооооооооооооооооооооооооооооооооо ...
```
:)

primus-anonymous commented on app/src/main/java/GeekBrians/Slava_5655380/UI/Fragments/NotesMetadataDisplayFragment.java:
```diff
@@ -19,13 +19,20 @@
     private Note selectedNote;
     private NotesDAO notes = new NotesDAO();
 
+    private void showNoteEditor(){
```
{ везде прямо поджата к скобке, код видимо надо форматнуть

SlavaGrishanin committed:
Empty commit

SlavaGrishanin committed:
Реализация сохранения заметки в локальное хранилище

SlavaGrishanin committed:
Реализация обновления NotesMetadataDisplayFragment после выхода из Editor

SlavaGrishanin committed:
Реализация обновления ContentDisplayActivity после выхода из Editor

SlavaGrishanin committed:
Реализация обновления NotesMetadataDisplayFragment после выхода из ContentDisplayActivity

SlavaGrishanin committed:
Исправление ошибки приводящей к вылету при выходе из Editor в ContentDisplay до нажатия на R.id.action_save

SlavaGrishanin committed:
Добавление NotesReadableAsSQLiteDB

SlavaGrishanin committed:
Реализация NotesSQLiteDBSource. Нужно исправить ошибку с ID

SlavaGrishanin committed:
Revert "Реализация NotesSQLiteDBSource. Нужно исправить ошибку с ID"

This reverts commit 090e0a9585487b5b52c9d8bb12ea0df87b10be48.

SlavaGrishanin committed:
Revert "Добавление NotesReadableAsSQLiteDB"

This reverts commit c26a8f525d27c66d0e72625434ad6a1d9f1f512a.

SlavaGrishanin committed:
Добавление NotesAsRoomDatabase

SlavaGrishanin committed:
Рефакторинг: вынесение открытия NotesReadableSource и NotesWritableSource в MainActivity

SlavaGrishanin committed:
Реализация NotesAsRoomDatabase

SlavaGrishanin committed:
Исправление ошибки приводящей к созданию новой заметки вместо обновления старой

SlavaGrishanin committed:
Исправление ошибки приводящей к повторному созданию экземпляра RoomDatabase

SlavaGrishanin committed:
Реализация NotesAsRoomDatabase в виде singleton класса

SlavaGrishanin committed:
Добавление контекстного меню note_metadata_browser_context_menu

SlavaGrishanin committed:
Реализация открытия NoteEditor через контекстное меню note_metadata_browser_context_menu

SlavaGrishanin committed:
Исправление ошибки приводящей к неправильной  инициализации .nid в конструкторе NoteRoomEntity(Note note, NotesDao notesDao)

SlavaGrishanin committed:
Реализация удаления заметки через контекстное меню note_metadata_browser_context_menu.
Исправление исправления ошибки приводящей к неправильной  инициализации .nid в конструкторе NoteRoomEntity(Note note, NotesDao notesDao) сделанном в коммите 4674662919bce2be5b1c9d4f558e4b14bb4d92a5

SlavaGrishanin committed:
Реализация возможности создать новую заметку

SlavaGrishanin committed:
Исправление реализации создания новой заметки

SlavaGrishanin committed:
Рефакторинг замена nid на uid

SlavaGrishanin committed:
Удаление класса NotesAsJSONFiles
